backend_test.go

```diff
@@ -452,9 +456,9 @@ func (e *Env) DeleteRole(t *testing.T) {
 func (e *Env) Login(t *testing.T) {
 	signingTime := time.Now()
 	signatureData := &signatures.SignatureData{
-		SigningTime: signingTime,
-		Role:        "test-role",
-		Certificate: e.TestCerts.InstanceCertificate,
+		SigningTime:            signingTime,
+		Role:                   "test-role",
+		CFInstanceCertContents: e.TestCerts.InstanceCertificate,
```

Comment:
I updated this field name to increase clarity around what it wants as well. In this file, there are actually _two_ certificates, and leaving it like it was could have been confusing for the user.